capstone-rs/src/capstone.rs

```diff
@@ -165,6 +165,37 @@ impl Capstone {
         }
     }
 
+    pub fn malloc<'a>(&'a self) -> *mut cs_insn {
+        unsafe { cs_malloc(self.csh()) }
+    }
+
+    pub fn free<'a>(&'a self, insn: *mut cs_insn) {
+        unsafe {
+            cs_free(insn, 1);
+        }
+    }
+
+    pub fn disasm_iter<'a>(
+        &'a self,
+        code: *mut *const u8,
```

Comment:
Avoid exposing raw pointers in the public API. We want to expose a memory-safe interface.

Reply:
Thank you for all your feedback. I really appreciate it.
I will take a look and see if I can clean even more the API. Feel free to decline this PR.